frontend/src/store/FamLoginUserState.ts

```diff
@@ -266,8 +267,12 @@ const cacheUserAccess = async () => {
     }
 };
 
-// TODO: check if login user needs to accept terms and conditions, if need, show terms here
-const requiresAcceptTermsCondition = async () => {};
+const requiresAcceptTermsCondition = async () => {
+    const needsToAccept = await AppActlApiService.userTermsAndConditionsApi.validateUserRequiresAcceptTermsAndConditions();
+    if (needsToAccept.data) {
+        showTermsForAcceptance();
+    }
```

Comment:
Can we just return a true/false and call "**showTermsForAcceptance**()" in the same "**handlePostLogin**" method and not here in state file?

Reply:
Hello! Sure can!